cdap-ui/app/cdap/components/AbstractWidget/WranglerEditor/wrangler-modal.scss

```diff
@@ -0,0 +1,289 @@
+/*
+ * Copyright © 2017 Cask Data, Inc.
```

Comment:
copyright year 2019. 

Is this a copy paste style?

Reply:
Yup 😐 Had to copy-paste the less file and convert it to scss file. Modified all syntax but missed the year in license.